packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -449,6 +449,115 @@ export const setupRDSClusterAndData = async (config: RDSConfig, queries?: string
   };
 };
 
+export const clearRDSClusterData = async (clusterInfo: ClusterInfo, region: string): Promise<void> => {
```

Comment:
nit: "cluster" isn't the interesting characteristic of the thing we're clearing; it's that we're accessing via Data API. Suggest renaming to reflect that, "clearTestDataUsingDataApi" or similar.

Reply:
Function name changed